.github/workflows/ci.yml

```diff
@@ -32,6 +33,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          cache: 'poetry'
```

Comment:
That broke CI a few month ago while not providing a significant speed increase. I decided to remove it: https://github.com/browniebroke/django-codemod/pull/658

```suggestion
```